src/components/Contacts/ContactFlow/contactFlowDefaultOptions.ts

```diff
@@ -61,37 +61,49 @@ export const getDefaultFlowOptions = (
       return [
         {
           id: uuidv4(),
-          name: getLocalizedPhase(t, PhaseEnum.Connection),
+          name:
+            phases?.find((phase) => phase?.id === PhaseEnum.Connection)?.name ||
```

Comment:
You are looping through `phases` 5 times. I bet you could loop through it once and get all these values to save computer processing